identify/extensions.py

```diff
@@ -113,6 +113,7 @@
     'md': {'text', 'markdown'},
     'mdx': {'text', 'mdx'},
     'mib': {'text', 'mib'},
+    'mjs': {'text', 'javascript'},
```

Comment:
should we additionally add `mjs` as a tag in case someone wants to target that specifically?

Reply:
Maybe wait for a use case? I found MDN actually argues against using the `.mjs` suffix since many tools don't support it: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#aside_%E2%80%94_.mjs_versus_.js . So it may not be particularly popular.